docs/fundamentals/code-analysis/style-rules/ide0003-ide0009.md

```diff
@@ -63,15 +81,14 @@ Me.capacity = 0
 capacity = 0
 ```
 
-## dotnet_style_qualification_for_property
+### dotnet_style_qualification_for_property
 
-|Property|Value|
-|-|-|
-| **Option name** | dotnet_style_qualification_for_property |
-| **Option values** | `true` - Prefer properties to be prefaced with `this.` in C# or `Me.` in Visual Basic<br /><br />`false` - Prefer properties _not_ to be prefaced with `this.` or `Me.` |
-| **Default option value** | `false` |
-
-### Example
+| Property                 | Value                                   | Description                                                                   |
+|--------------------------|-----------------------------------------|-------------------------------------------------------------------------------|
+| **Option name**          | dotnet_style_qualification_for_property |                                                                               |
+| **Option values**        | `true`                                  | Prefer properties to be prefaced with `this.` in C# or `Me.` in Visual Basic. |
+|                          | `false`                                 | Prefer properties _not_ to be prefaced with `this.` or `Me.`.                 |
+| **Default option value** | `false`                                 |                                                                               |
```

Comment:
I'm not sure if this is correct or not. Can you verify whether `false` accepts both `this.MyProperty` and `MyProperty` or it only accepts `MyProperty`?

Reply:
IDE0009 (qualifiers must be present) accepts both forms when this option is `false`.